src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -237,6 +248,65 @@ void Pooling::getSupportedDescriptors() {
     InferenceEngine::Precision inputPrecision = getOriginalInputPrecisionAtPort(0);
     InferenceEngine::Precision outputPrecision = getOriginalOutputPrecisionAtPort(0);
 
+#if defined(OV_CPU_WITH_ACL)
```

Comment:
jist wondering can we implemet code block below as static function of ACLPoolingExecutor and reuse here and inside init() method?

Reply:
Implemented